apps/server/src/modules/user-login-migration/service/school-migration.service.spec.ts

```diff
@@ -53,17 +54,23 @@ describe('SchoolMigrationService', () => {
 		const schoolDO: SchoolDO = schoolDOFactory.buildWithId({
 			id: 'schoolId',
 			name: 'schoolName',
-			officialSchoolNumber: '3',
+			officialSchoolNumber: 'officialSchoolNumber',
```

Comment:
Please refactor to use our test structure.

Reply:
even if they use the same test data ?